doozerlib/brew.py

```diff
@@ -393,13 +393,20 @@ class KojiWrapper(koji.ClientSession):
           will return the cached value.
     """
 
-    koji_wrapper_lock = threading.Lock()
-    koji_call_counter = 0  # Increments atomically to help search logs for koji api calls
-    # Used by the KojiWrapper to cache API calls, when a call's args include a KojiWrapperOpts with caching=True.
-    # The key is either None or a brew event id to which queries are locked. The value is another dict whose
-    # key is a string respresentation of the method to be invoked and the value is the cached value returned
-    # from the server.
-    koji_wrapper_result_cache = {}
+    """
+    If caching should be enabled for all uses of this class. This is generally
+    not recommended unless you are trying to record API calls for testing purposes.
+    """
+    force_global_caching: bool = False
```

Comment:
It's not used?

Reply:
Just work I was doing while tests ran :) . More on it in the next commit, but not presently used at all.